ethereum-consensus/src/configs/holesky.rs

```diff
@@ -0,0 +1,88 @@
+use ssz_rs::U256;
+
+use crate::{
+    configs::Config,
+    primitives::{Epoch, ExecutionAddress, Gwei, Version, FAR_FUTURE_EPOCH},
+};
```

Comment:
let's keep imports all together

```suggestion
use ssz_rs::U256;
use crate::{
    configs::Config,
    primitives::{Epoch, ExecutionAddress, Gwei, Version, FAR_FUTURE_EPOCH},
};
```